starwars/src/App.js

```diff
@@ -33,6 +34,10 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <div>{this.state.starwarsChars.map((x)=>{
```

Comment:
try to name arguments / variables a little more verbose to denote what they actually are. single letter variables are ok in math but not so good when doing software engineering. iterators have convention from older languages such as i j k etc; but for modern software engineering. the use of more verbosity is considered good software engineering practice